pgex/examples/glow_effect/main.py

```diff
@@ -0,0 +1,93 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import asyncio
+import math
+from typing import Tuple
+
+import pygame
+
+Point = Tuple[int, int]
+
+
+def distance(p1: Point, p2: Point) -> float:
+    dx = p1[0] - p2[0]
+    dy = p1[1] - p2[1]
+    radicand = dx**2 + dy**2
+    return math.sqrt(radicand)
```

Comment:
Likely can be entirely substituted with `pygame.Vector2.distance_to` (also moved to the same file as the class, but likely the function is not even necessary, so can be simply handled in the `update` method, see comment below)